### DISCUSSION TITLE
为什么PCL2无缘无故提示我有java正在运行，不能删除mod？

### DISCUSSION THREAD

**Discussion by LWCLWCNB** [Meloong-Git/PCL#1729] [提问] [answered]
玩我的世界时，当我想删一个模组时PCL2无缘无故说有java正在运行，但是任务管理器里却并没有java在运行

**Comment by Chiloven945** [comment, marked as answer]
请将游戏关闭后再尝试增删模组

**Comment by IF-desu** [comment]
关闭游戏再进行操作。。。如果还是不行尝试重启pcl或者电脑

**Reply by IF-desu** [reply]
其实可以尝试手动删除mod文件，如果资源管理器不提示文件正在使用大概率是pcl的锅，发条issue或许可以

**Reply by Chiloven945** [reply]
> 其实可以尝试手动删除mod文件，如果资源管理器不提示文件正在使用大概率是pcl的锅，发条issue或许可以

一般都不会是PCL的问题，有时候游戏会占用mod估计读取什么资源文件什么的，PCL最多就在mod管理界面的时候占用

**Comment by LWCLWCNB** [comment]
解决了，谢谢
